index.d.ts

```diff
@@ -153,14 +153,40 @@ export interface RetryOptions {
 /**
 Options are the same as `window.fetch`, with some exceptions.
 */
-export interface Options extends RequestInit {
+export interface Options extends Omit<RequestInit, 'headers'> {
 	/**
 	HTTP method used to make the request.
 
 	Internally, the standard methods (`GET`, `POST`, `PUT`, `PATCH`, `HEAD` and `DELETE`) are uppercased in order to avoid server errors due to case sensitivity.
 	*/
 	method?: LiteralUnion<'get' | 'post' | 'put' | 'delete' | 'patch' | 'head', string>;
 
+	/**
+	Override default Headers
+
+	Extra types:
+	- `{[key: string]: undefined}` for add ability to remove headers with extend()
+
+	@example
+	```
+	import ky from 'ky';
+
+	const original = ky.create({
+		headers: {
+			rainbow: 'rainbow',
+			unicorn: 'unicorn'
+		}
+	});
+
+	const extended = original.extend({
+		headers: {
+			rainbow: undefined
+		}
+	});
```

Comment:
You need to show the resulting headers as a comment to make it clear what's happening.

Reply:
I think I was clear what needs to change here. And please don't copy-paste a generic comment.